tests/test_api.py

```diff
@@ -136,7 +136,12 @@ def test_compact_json(self) -> None:
         )
 
     def test_read_write_read_compare(self) -> None:
-        for metadata in [Root.type, Snapshot.type, Timestamp.type, Targets.type]:
+        for metadata in [
+            Root.type,
+            Snapshot.type,
+            Timestamp.type,
+            Targets.type,
+        ]:
```

Comment:
it's a nitpick but in cases like this and the one below a bit of tweaking could save ten lines of code:

This probably fits and is a better name:
`        for role in [Root.type, Snapshot.type, Timestamp.type, Targets.type]:`
or :
`        for rolename in TOP_LEVEL_ROLE_NAMES:`

Maybe this one is not worth modifying the PR for but this sort of thing adds up... and makes the tests harder to read

Reply:
`for role in [Root.type, Snapshot.type, Timestamp.type, Targets.type]:` is more than 80 characters, but
`for rolename in TOP_LEVEL_ROLE_NAMES:` will do the work.
I thought we have deleted `TOP_LEVEL_ROLE_NAMES` when we introduce the constatnts and it's a nice surprise we didn't.